overviews/quasiquotes/hygiene.md

```diff
@@ -10,11 +10,11 @@ outof: 13
 ---
 **Denys Shabalin, Eugene Burmako** <span class="label warning" style="float: right;">EXPERIMENTAL</span>
 
-The notion of hygiene has been widely popularized by macro research in Scheme. A code generator is called hygienic if it ensures absence of name clashes between regular and generated code, preventing accidental capture of identifiers. As numerous experience reports show, hygiene is of great importance to code generation, because name binding problems are often very non-obvious and lack of hygiene might manifest itself in subtle ways.
+The notion of hygiene has been widely popularized by macro research in Scheme. A code generator is called hygienic if it ensures the absence of name clashes between regular and generated code, preventing accidental capture of identifiers. As numerous experience reports show, hygiene is of great importance to code generation, because name binding problems are often non-obvious and lack of hygiene might manifest itself in subtle ways.
 
-Sophisticated macro systems such as Racket's have mechanisms that make macros hygienic without any effort from macro writers. In Scala we don't have automatic hygiene yet - both of our codegen facilities (compile-time codegen with macros and runtime codegen with toolboxes) require programmers to handle hygiene manually. Fixing this is our number one priority for 2.12 (see [future prospects](/overviews/quasiquotes/future.html)), but in the meanwhile you need to know how to work around the absence of hygiene, which is what this section is about.
+Sophisticated macro systems such as Racket's have mechanisms that make macros hygienic without any effort from macro writers. In Scala we don't have automatic hygiene yet - both of our codegen facilities (compile-time codegen with macros and runtime codegen with toolboxes) require programmers to handle hygiene manually. Fixing this is our number one priority for 2.12 (see [future prospects](/overviews/quasiquotes/future.html)) but, in the meantime, you must know how to work around the absence of hygiene, which is what this section is about.
```

Comment:
The `Fixing this is our number one priority for 2.12...` sentence is out of date too and should be removed altogether.